packages/monaco/src/browser/monaco-theming-service.ts

```diff
@@ -202,4 +203,12 @@ export class MonacoThemingService {
         return str;
     }
 
+    private cleanEmpty(obj: any): void {
+        for (const key in obj) {
+            // eslint-disable-next-line no-null/no-null
+            if ([null, undefined].includes(obj[key])) {
+                delete obj[key];
```

Comment:
This deletes the key from the original object completely, is it ok to do so or should we create a copy?

Reply:
if the idea is that returned `json` in line 121 won't have those then why should we create a copy?

Reply:
Doubled checked and since the object is created line 107 it means it's fine indeed. Just wanted to be sure.

Reply:
> then why should we create a copy

I didn't know if that struct could be safely mutated from looking at the change if some clients want to keep the `null` value or not, hence why I asked btw.